pkg/nrtupdater/node_getter.go

```diff
@@ -39,7 +31,7 @@ type DisabledNodeGetter struct {
 }
 
 func (ng *DisabledNodeGetter) Get(ctx context.Context, nodeName string, opts metav1.GetOptions) (*corev1.Node, error) {
-	return nil, NotConfigured{}
+	return nil, fmt.Errorf("%w", NotConfigured)
```

Comment:
it seems to me you can just return `NotConfigured` here

Reply:
Yes, I did it that way following one of the sentences in the article about errors
"""
If a function is defined as returning an error wrapping some sentinel or type, do not return the underlying error directly.
```
var ErrPermission = errors.New("permission denied")

// DoSomething returns an error wrapping ErrPermission if the user
// does not have permission to do something.
func DoSomething() error {
    if !userHasPermission() {
        // If we return ErrPermission directly, callers might come
        // to depend on the exact error value, writing code like this:
        //
        //     if err := pkg.DoSomething(); err == pkg.ErrPermission { … }
        //
        // This will cause problems if we want to add additional
        // context to the error in the future. To avoid this, we
        // return an error wrapping the sentinel so that users must
        // always unwrap it:
        //
        //     if err := pkg.DoSomething(); errors.Is(err, pkg.ErrPermission) { ... }
        return fmt.Errorf("%w", ErrPermission)
    }
    // ...
}
```
"""

Reply:
ok, good find. TIL.